ReportUtilityBills/tests/utility_bills_test.rb

```diff
@@ -1110,23 +1115,36 @@ def _load_timeseries(pv_size_kw, use_hourly_electricity)
     return fuels
   end
 
-  def _bill_calcs(fuels, header, pv_systems, utility_bill_scenario)
+  def _bill_calcs(fuels, header, pv_systems, utility_bill_scenario, monthly_data)
     runner = OpenStudio::Measure::OSRunner.new(OpenStudio::WorkflowJSON.new)
-    output_format = 'csv'
-    output_path = File.join(File.dirname(__FILE__), "results_bills.#{output_format}")
+    args = { output_format: 'csv', include_annual_bills: true, include_monthly_bills: true }
 
     utility_rates, utility_bills = @measure.setup_utility_outputs()
     @measure.get_utility_rates(@hpxml_path, fuels, utility_rates, utility_bill_scenario, pv_systems)
     @measure.get_utility_bills(fuels, utility_rates, utility_bills, utility_bill_scenario, header)
 
-    @measure.report_runperiod_output_results(runner, utility_bills, output_format, output_path, utility_bill_scenario.name)
+    # Annual
+    output_path = File.join(File.dirname(__FILE__), "results_bills.#{args[:output_format]}")
+    @measure.report_runperiod_output_results(runner, args, utility_bills, output_path, utility_bill_scenario.name)
 
     # Check written values exist and are registered
     assert(File.exist?(@bills_csv))
     actual_bills = _get_actual_bills(@bills_csv)
 
     _check_for_runner_registered_values(runner, nil, actual_bills)
 
+    # Monthly
+    @measure.get_monthly_output_results(args, utility_bills, utility_bill_scenario.name, monthly_data)
+
+    # Check written values exist and are registered
+    actual_monthly_bills = _get_actual_monthly_bills(monthly_data)
+
+    # Check sum of monthly equal to annual
+    actual_bills.keys.each do |bill|
+      assert(actual_monthly_bills.keys.include?(bill))
+      assert_in_delta(actual_bills[bill], actual_monthly_bills[bill].sum, 0.1) # within 10 cents
+    end
```

Comment:
I ran various test files and noticed that the `base-simcontrol-runperiod-1-month.xml file` has all zeroes in the monthly bills output file. Can you make sure this test catches the error in that file, and then fix it?